ASM/training-mode/Onscreen Display/Act OoJumpsquat.asm

```diff
@@ -77,23 +22,23 @@
     # Check For Not JumpSqaut
     lwz r3, 0x10(playerdata)
     cmpwi r3, 0x18
-    beq Exit
```

Comment:
`Moonwalk_Exit` is used elsewhere, but it's just a confusing/silly name for exit. Can you revert these renames.

Reply:
you know what nvm

Reply:
Thank you for merging this.
Sorry, I'll using `Exit` when I touch it next time.